<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FreeTypeBatch</class>
 <widget class="QDialog" name="FreeTypeBatch">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>357</width>
    <height>181</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FreeType Import Options</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Render Point &amp;Sizes:</string>
       </property>
       <property name="buddy">
        <cstring>fPointSizes</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="fPointSizes"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Common Settings</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="2" colspan="2">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QRadioButton" name="fMonochrome">
          <property name="text">
           <string>&amp;Monochrome</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="fGrayscale">
          <property name="text">
           <string>&amp;Grayscale</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Screen &amp;Resolution:</string>
        </property>
        <property name="buddy">
         <cstring>fResolution</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Max &amp;Character:</string>
        </property>
        <property name="buddy">
         <cstring>fMaxChar</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QSpinBox" name="fMaxChar">
        <property name="maximum">
         <number>1114111</number>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSpinBox" name="fResolution">
        <property name="maximum">
         <number>65535</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Font &amp;Name:</string>
       </property>
       <property name="buddy">
        <cstring>fFontName</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="fFontName"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>FreeTypeBatch</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>FreeTypeBatch</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
